src/README.md

```diff
@@ -99,6 +99,18 @@ Tables of named input and output links.  These tables are initialized by
 the engine for use in processing and are *read-only*.
 
 
+— Field **myapp.appname**
+
+Name of the app. *Read-only*.
+
+
+— Method **myapp:start**
+
+*Optional*. Start the app. At this the `input` and `output` link tables are
+populated, and `appname` is set. The app can perform additional initialization
+if required.
```

Comment:
The details of this callback are surprising to me.

Based on the name `start` I would expect the callback to be run once when an app instance is first created and before any traffic is processed. Looking at the code it seems like `start()` is actually always called, on every app instances, each time you run `engine.configure()` (even if your new config is identical to the old one).

Could you please update either the name, the description, or the implementation to make the behavior & appropriate uses more obvious?